src/components/ReplyComment/hooks/useReplyComment.js

```diff
@@ -179,12 +182,8 @@ const useReplyComment = ({itemProp, indexFeed, dataFeed, updateParent, page, get
       }
     }
   };
-  const createComment = async (isAnonimity, anonimityData) => {
-    let sendPostNotif = false;
-    if (page !== 'DetailDomainScreen') {
-      sendPostNotif = true;
-    }
 
+  const handleAddedComment = (isAnonimity, anonimityData) => {
     const commentWillBeAddedData = {
       ...defaultData,
       data: {...defaultData.data, text: textComment}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [185-208]

The function `createComment` has been refactored into two separate functions: `handleAddedComment` and `createComment`. This improves modularity and readability of the code. However, ensure that all instances where `createComment` was previously called have been updated to call these two functions as needed.

```diff
+ const handleAddedComment = (isAnonimity, anonimityData) => {
+   const commentWillBeAddedData = {
+     ...defaultData,
+     data: {...defaultData.data, text: textComment}
+   };
+   setTemporaryText('');
+   setNewCommentList([...newCommentList, commentWillBeAddedData]);
+ };
+
+ const createComment = async (isAnonimity, anonimityData) => {
+   let sendPostNotif = false;
+   if (page !== 'DetailDomainScreen') {
+     sendPostNotif = true;
+   }
+   handleAddedComment(isAnonimity, anonimityData);
```

228:
The `scrollToEnd` function is now called with an options object `{animated: true}`. This will cause the scroll view to animate scrolling to the end. Ensure this change in behavior is intended.

```diff
- scrollViewRef.current.scrollToEnd();
+ scrollViewRef.current.scrollToEnd({animated: true});
```

291:
The `curHeight` state variable has been added to the exported object. Make sure that it is used correctly wherever this object is imported.

```diff
+ curHeight,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->